.github/workflows/build-check.yml

```
@@ -0,0 +1,25 @@
+name: Build Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
```

Same suggestion re: the Actions linter

```suggestion
      uses: actions/setup-node@v3
```